content/applications/essentials/contacts.rst

```diff
@@ -204,6 +196,9 @@ slide the :guilabel:`Unpublished` toggle to the active, :guilabel:`Published` po
 
 Repeat these steps for all partners.
 
+.. seealso::
+   :doc:`Resellers <../sales/crm/track_leads/resellers>`
+
```

Comment:
Instead of re-explaining publishing partners (in the paragraphs above the seealso), just link to the new _Resellers_ documentation
```suggestion
.. seealso::
   Follow the :doc:`Resellers documentation <../sales/crm/track_leads/resellers>` for more
   information on publishing partners on the website.
   
```